src/pages/Home/Home.page.tsx

```diff
@@ -1,6 +1,30 @@
+import pizzaLetter from "../../utils/pizza-letters.jpg";
+import BackgroundImg from "../../utils/ProperHomePageBackgroundPic.jpg";
+
 const HomePage = () => {
-  return <div>Homepage && Salutare de la Andrei 1</div>;
+  return (
+    <div>
+      <div
+        className="h-screen w-full text-white bg-center bg-cover  bg-scroll flex flex-col justify-end  "
```

Comment:
Replace bg-scroll with bg-fixed and remove it from inline-styling

Reply:
Replaced and removed